src/DataStructures/Tensor/Expressions/Contract.hpp

```diff
@@ -585,6 +585,15 @@ struct TensorContract
     }
   }
 
+  /// \brief Get a component from a `Tensor` in this expression's subtree of the
```

Comment:
Wrong docs?

Reply:
Can you explain what you mean by your question? I'm so sorry, I'm not following

Reply:
I understand now, I guess the function just doesn't seem useful based on the documentation. Returning a random tensor component that you don't know what it is seems a bit silly. I think it's better to return the size.